layers/+spacemacs/spacemacs-project/config.el

```diff
@@ -0,0 +1,21 @@
+;;; config.el --- Projectile Layer Configuration File for Spacemacs
+;;
+;; Copyright (c) 2012-2018 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+
+;; Layer Variables
+
+(defvar projectile-keymap-prefix (kbd "C-c C-p")
```

Comment:
This should be a `setq` I would think. That said, I'd think that something like https://github.com/ericdanan/counsel-projectile/pull/92 is the right fix. Then the decision as to what the projectile keymap prefix is in spacemacs (if any) is a separate decision. Maybe it should be `C-c C-p`, but I don't know how @syl20bnr thinks about things like this.

Reply:
@seagle0128 could you elaborate? I don't see a commit in projectile or spacemacs setting the prefix to `C-c p`, but maybe I'm missing something?